Mage/src/main/java/mage/abilities/costs/OrCost.java

```diff
@@ -90,8 +94,7 @@ public boolean pay(Ability ability, Game game, Ability source, UUID controllerId
                         Outcome.Detriment, sb.toString(), null,
                         CardUtil.getTextWithFirstCharUpperCase(usable.get(0).getText()),
                         CardUtil.getTextWithFirstCharUpperCase(usable.get(1).getText()),
-                        ability, game
-                )) {
+                        ability, game)) {
```

Comment:
There is a risk for heavy changes in big files if your IDE is auto formatting those kind of line breaks.

Reply:
I'm no Vscode config expert, but aparently you can format only on modifications and not whole file by adding that to your config file:
`"editor.formatOnSaveMode": "modifications"`

Reply:
Thanks, I've changed to that hopefully it helps.